tests/integration/test_fim/test_files/test_ambiguous_confs/test_ignore_works_over_restrict.py

```diff
@@ -164,35 +168,38 @@ def test_ignore_works_over_restrict(folder, filename, triggers_event, tags_to_ap
 
     tags:
         - scheduled
-        - time_travel
     '''
     logger.info('Applying the test configuration')
     check_apply_test(tags_to_apply, get_configuration['tags'])
-    scheduled = get_configuration['metadata']['fim_mode'] == 'scheduled'
 
     # Create file that must be ignored
     logger.info(f'Adding file {os.path.join(testdir1, filename)}, content: ""')
     create_file(REGULAR, folder, filename, content='')
 
-    # Go ahead in time to let syscheck perform a new scan if mode is scheduled
-    logger.info(f'Time travel: {scheduled}')
-    check_time_travel(scheduled, monitor=wazuh_log_monitor)
+    # Timeout for FIM to generate the log.
+    test_timeout = 5  # seconds
```

Comment:
Change the variable name to a representative name.

Reply:
This is fixed in commit [5eb3235](https://github.com/wazuh/wazuh-qa/pull/3078/commits/5eb3235481caf0ea3699bb83aea4a028819101bb)